lib/beacon/pub_sub.ex

```diff
@@ -1,39 +1,36 @@
 defmodule Beacon.PubSub do
   @moduledoc false
 
-  alias Beacon.Content.Component
-  alias Beacon.Content.ErrorPage
-  alias Beacon.Content.Layout
-  alias Beacon.Content.LiveData
-  alias Beacon.Content.LiveDataAssign
-  alias Beacon.Content.Page
+  alias Beacon.Content
```

Comment:
I'm not a big fan of partial aliases, as they make it harder to grep the codebase for specific modules.  In here we don't even use the `Beacon.Content` module, so it also gives a different expectation for the module's dependencies at a glance.

If we used
```elixir
alias Beacon.Content.Layout
alias Beacon.Content.Page
```
that should be all we need for full aliasing here after the old code removal

Reply:
> I'm not a big fan of partial aliases, as they make it harder to grep the codebase for specific modules.  In here we don't even use the `Beacon.Content` module, so it also gives a different expectation for the module's dependencies at a glance.
> 
> 
> 
> If we used
> 
> ```elixir
> 
> alias Beacon.Content.Layout
> 
> alias Beacon.Content.Page
> 
> ```
> 
> that should be all we need for full aliasing here after the old code removal

That's a good point, I'm gonna change it before merging.